linkage/pvs_like_case_study/generate_simulated_data/generate_simulated_data_small_sample.ipynb

```diff
@@ -0,0 +1,2894 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #32.    # What to do about ground truth here? This simple linkage could already be introducing errors, since SSN is not without noise!
</pre><p>perhaps alter your custom config to have no noise on SSN here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research_prl/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
In order to fully eliminate this problem, we'd have to make SSN noiseless not only in SSA, but also in taxes, which doesn't seem right. I've handled this in a more sophisticated way in the next version of the case study (will open PR soon).